pkg/cloudfoundry/YamlUtil.go

```diff
@@ -0,0 +1,236 @@
+package cloudfoundry
```

Comment:
does this belong into the `cloudfoundry` package?

Isn't this rather general so that we could put it into e.g. `pkg/yaml`

Reply:
I had similar thoughts and since that is used with a clear cloudfoundry only scope in groovy I decided for having that also here in a `cloudfoundry` package. I'm not sure if that here is generic enough for being in a more generic `yaml` package. E.g. we have the format that we replace parameter espaced like this `((param))`.

@OliverNocon : In case that is generic enough for being in a yaml package I can move it.

@OliverNocon : do you still think it is generic enought?

Reply:
I would vote for a generic package (pkg/yaml), if it is actually generic.

Reply:
> I would vote for a generic package (pkg/yaml), if it is actually generic.

I would also vote for a generic package if it actually generic. But I'm asking me if it is generic (open question, not a rhetoric question here). I guess everybody agrees having that in a generic package if it is generic ...

Reply:
Maybe it is not yet completely generic, but it could well be developed to be. On the go, as needed. We have just now discussed that we will need something like this for some step.

Reply:
Will move it into a generic package.

My question is mostly about the `((param))` syntax. Is that something which is cloudfoundry specific (e..g. does the cf cli somehow support that) or is that more a common way to handle params in yaml? Basically I can't find something related to a parameter syntax (`((param))`) when searching in the web. Maybe that was choosen by "us"  in favour of something like `${param}` in order to keep the yaml file (better)  parsable when the parameters (unresolved) are in the file.

Reply:
I have to admit that I don't know where it comes from.
Syntax-wise we ideally go with something which is known rather than something which is too special.

`${param}` is definitely well known.
Maybe we could it evolve into such a direction at some point. I am on purpose NOT asking for doing the changes now :-)